macro/run_simScript.py

```
@@ -545,6 +612,19 @@
     else:
         # point source
         myPgun.SetXYZ(options.Vx * u.cm, options.Vy * u.cm, options.Vz * u.cm)
+    if options.histoFile:
+        if not len(options.histoFile) == len(options.histoName):
+            raise ValueError("Must have the same number of specified histogram files")
+        if not len(options.histoX) == len(options.histoFile):
+            raise ValueError("Must at least specify one histogram variable X variable per histogram!")
+
+        for _hf, _idx in zip(options.histoFile, range(len(options.histoFile))):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)